scripts/report_scripts/qc_report_per_sample.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: "SARS-CoV-2 QC report"                                                                                                                                                                 
+date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
+params:
+    @PARAMS_SAMPLE_NAME@
+
+---
+
+Read config.yml for the locations of sample sheet, pipeline output and coverage_file. 
+```{r, child="_settings.Rmd"}
+```
+
+```{r setup, include=FALSE, message = FALSE, warning = FALSE}
+knitr::opts_chunk$set(echo = TRUE, message = FALSE, warning = FALSE)
+knitr::opts_knit$set(root.dir = params$workdir)
+```
+
+# Input Settings
+```{r InputSettings, echo = FALSE, include = FALSE}
+sample_name <- sampleName
+coverage_file <- paste0(coverage_dir,"/",sample_name,"_merged_covs.csv")
+``` 
+
+```{r printInputSettings, echo = FALSE, include = FALSE}
+# coverage_file <- 'tests/coverage/Test_merged_covs.csv'
+# sample_name <- 'Test'
+``` 
+### Sample: `r sample_name 
+```{r reading data, include = FALSE}
+coverage.df <- read.table(coverage_file,sep = "\t", header = T) 
+coverage.df <- na_if(coverage.df, "[]")
+coverage.df <- gsub('\\[','',coverage.df)
+coverage.df <- gsub('\\]','',coverage.df)
+# fixme: this can be sure done in a nicer and more compact way
```

Comment:
Yes.

Reply:
thats something in Vics script i would not like to intervene there

Reply:
Ah, right, you moved that file.  Never mind then!